packages/verifiable-credentials/test/create-credentials.test.ts

```diff
@@ -0,0 +1,95 @@
+// https://nodejs.org/api/assert.html
+const assert = require('assert');
+import { Utils } from '@verida/client-ts';
+import Credentials from '../src/credentials';
+import SharingCredential from '../src/sharing-credential';
+import { config, connect } from './config'
+
+
+describe('Credential tests', function () {
+
+    describe('Credential Units', function () {
+        this.timeout(100000);
+        let appContext;
+        let credential;
+
+        beforeEach(async function () {
+            appContext = await connect(config.PRIVATE_KEY_1);
+
+            credential = new Credentials(appContext)
+        });
+
+        it('Verify Credential JWT was created correctly', async function () {
+
+            const jwt: any = await credential.createCredentialJWT(config.SUBJECT_DID, config.CREDENTIAL_DATA);
+
+            const issuer = await credential.createIssuer();
+
+            // Decode the credential
+            const decodedCredential = await credential.verifyCredential(jwt.didJwtVc)
+
+            // Obtain the payload, that contains the verifiable credential (.vc)
+            const payload = decodedCredential.payload
+            const vc = payload.vc
+
+
+            // Verify the "Payload"
+            assert.equal(payload.iss, issuer.did, 'Credential issuer matches expected DID')
+
+            // Verify the "Verifiable Credential"
+            assert.deepEqual(vc.credentialSubject, config.CREDENTIAL_DATA, 'Credential data is valid');
+            assert.deepEqual(issuer.did, vc.issuer, 'Issuer matches expected DID');
+            assert.equal(vc.credentialSchema.id, config.CREDENTIAL_DATA.schema, 'Credential schema is correct')
+            assert.equal(vc.sub, config.SUBJECT_DID, 'Credential subject matches expected DID')
+
+            // Verify the data matches the schema
+            const record = vc.credentialSubject
+            const schema = await appContext.getClient().getSchema(record.schema)
+            const isValid = await schema.validate(config.CREDENTIAL_DATA);
+            assert.equal(true, isValid, 'Credential subject successfully validates against the schema');
+
+            // Note: Don't need to check the signature, because the did-jwt-vc does this for us
+
+            // @todo: confirm vc.issuanceDate was within the last 10 seconds
```

Comment:
Unsure about this - see previous comment about `issuanceDate`

Reply:
> @todo: confirm vc.issuanceDate was within the last 10 seconds


@tahpot should I open a backlog issue for this?

Reply:
@cmcWebCode40 Yes.

In light of the above, probably need two test cases. Test `issuanceDate` is within 10 seconds, if no `issuanceDate` provided. Otherwise do a test with a user supplied `issuanceDate`.

Reply:
Backlog issue for this change https://github.com/verida/verida-js/issues/148